src/login/inhibit.c

```diff
@@ -137,7 +138,7 @@ static void help(void) {
                "     --why=STRING         A descriptive string why is being inhibited\n"
                "     --mode=MODE          One of block or delay\n"
                "     --list               List active inhibitors\n"
-               , program_invocation_short_name);
+               , short_name);
```

Comment:
`program_invocation_short_name` is a glibc specific global. Instead of changing code that is normally compiled against glibc, I'd prefer adding a substitution to musl_missing.h (or basic/missing.h) if `__GLIBC__` is undef.
This would then just need a call to setting function with `argv[0]` from each `main()`, or better `log_parse_environment()`, as that is called in each `main()` already, to set `program_invocation_short_name`.